python/google/protobuf/json_format.py

```diff
@@ -88,7 +88,8 @@ class ParseError(Error):
 
 def MessageToJson(message,
                   including_default_value_fields=False,
-                  preserving_proto_field_name=False):
+                  preserving_proto_field_name=False,
+                  omitting_insignificant_whitespace=False):
```

Comment:
@anandolee: It would be nice, but I think it should have the same behavior as in languages where support already exists.
Java example: [constructor](https://github.com/google/protobuf/blob/master/java/util/src/main/java/com/google/protobuf/util/JsonFormat.java#L134) and [default usage (`JsonFormat.printer()`)](https://github.com/google/protobuf/blob/master/java/util/src/main/java/com/google/protobuf/util/JsonFormat.java#L106-L108)